app/src/main/java/tech/ula/utils/AndroidUtility.kt

```diff
@@ -68,11 +68,14 @@ fun getBranchToDownloadAssetsFrom(assetType: String): String {
     }
 }
 
-fun getAvailableStorageInMB(): Long {
-    val bytesInMB = 1048576
-    val stat = StatFs(Environment.getExternalStorageDirectory().path)
-    val bytesAvailable = stat.blockSizeLong * stat.availableBlocksLong
-    return bytesAvailable / bytesInMB
+class StorageUtility(statFs: StatFs) {
```

Comment:
You can declare `statFs` as a member property of the class in the constructor by just doing `class StorageUtility(private val statFs: StatFs)`